client/me/purchases/list/site/index.jsx

```diff
@@ -34,7 +34,9 @@ const PurchasesSite = ( { isPlaceholder, name, purchases, slug, domain } ) => {
 	return (
 		<div className={ classNames( 'purchases-site', { 'is-placeholder': isPlaceholder } ) }>
 			<SectionHeader label={ label }>
-				<span className="purchases-site__slug">{ domain }</span>
+				{ isDomainOnly && (
```

Comment:
Sorry, I removed it when testing and forgot to update it! Thanks for fixing this.